projects/core/components/primitive-textfield/value-decoration/value-decoration.template.html

```diff
@@ -1,4 +1,4 @@
 <span #pre subtree characterData waMutationObserver>{{prefix}}</span>
-<span class="t-ghost">{{value}}</span>
 <span class="t-filler">{{filler}}</span>
```

Comment:
Filler has to go after ghost:
![image](https://user-images.githubusercontent.com/11832552/141812440-4bfb3659-1c97-4315-a8a7-e1275e7198f6.png)

You can still achieve your desired CSS effect with `~` instead of `+`.